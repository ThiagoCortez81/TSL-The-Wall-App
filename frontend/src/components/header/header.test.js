import React from 'react';
import { shallow } from 'enzyme';
import Header from './header';
describe('Auth', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Header debug/>);

        expect(component).toMatchSnapshot();
    });
});