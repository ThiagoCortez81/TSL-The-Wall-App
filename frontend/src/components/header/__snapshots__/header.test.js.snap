// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth should render correctly in "debug" mode 1`] = `
<Uncontrolled(Navbar)
  bsStyle="default"
  className="bg-primary"
>
  <NavbarHeader>
    <NavbarBrand>
      <NavLink
        activeClassName="active"
        aria-current="page"
        to="/"
      >
        The Wall
      </NavLink>
    </NavbarBrand>
  </NavbarHeader>
  <Nav
    bsClass="nav"
    className="float-right"
    justified={false}
    pullLeft={false}
    pullRight={false}
    stacked={false}
  >
    <li>
      <NavLink
        activeClassName="active"
        aria-current="page"
        className=""
        to="/"
      >
        Home
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="active"
        aria-current="page"
        to="/profile"
      >
        My profile
      </NavLink>
    </li>
  </Nav>
</Uncontrolled(Navbar)>
`;
